src/snowflake/snowpark/_internal/open_telemetry.py

```diff
@@ -30,8 +39,8 @@ def open_telemetry_context_manager(func, dataframe):
     if open_telemetry_found:
         class_name = func.__qualname__
         name = func.__name__
-        tracer = (
-            trace.get_tracer(f"snow.snowpark.{class_name.split('.')[0].lower()}")
+        tracer = trace.get_tracer(
+            f"snow.snowpark.{class_name.split('.')[0].lower()}"
```

Comment:
snow or snowflake?

Reply:
it is snow, this is a naming tradition from telemetry team and they want to keep it the same